cms/views.py

```diff
@@ -49,7 +49,7 @@
 
 if DJANGO_2_2:
```

Comment:
Is this a bad variable name where it means greater than 2.2?

I'm just thinking that while doing version support, we maybe can remove this check or name things better

Reply:
I think it needs to go. `DJANGO_2_2` means Django version 2.2 or lower.